www/controllers/curate.js

```diff
@@ -63,13 +67,17 @@ define(['JBrowse/Browser']
             var data = {
                 type:  'correction',
                 value: normalizeFeatures(transcripts)
-            }
+            };
             return http.post('data/tasks/' + id, data).then(function (response) {
                 localStorage.clear();
                 console.log('saved submission');
-            });
-            // what on failure?
-        }
+                },
+                // invoked when request fails (or done button fails)
+                function (response) {
+                    $.confirmDialog(false);
```

Comment:
Yeah, it isn't really required since we mention in generic message what is saved and what not. However, the special message for this scenario is still in the place.

I'll update the files.